maintainers/maintainer-list.nix

```diff
@@ -9612,6 +9612,12 @@
     githubId = 6375438;
     keys = [ { fingerprint = "5214 2D39 A7CE F8FA 872B  CA7F DE62 E1E2 A614 5556"; } ];
   };
+  govindsi = {
+    email = "govind.sk85@gmail.com";
+    github = "govindsi";
+    githubId = 8924200;
+    name = "Govind Singh";
+  };
```

Comment:
Per CONTRIBUTING.md this should be done in a separate commit.

Reply:
fixed with https://github.com/NixOS/nixpkgs/pull/458358/commits/851feddc38589b5768c1494b0443983b3fc43967